about.html

```diff
@@ -0,0 +1,104 @@
+<body>
+  <div id="background">
+   <header>
+    <nav id="mobile-nav">
+    	<i id="menu-btn" class="fas fa-bars fa-2x"></i>
+      <ul id="mobile-list" class="dn">
+      	<li class="mobile-list-item"><a href="./">Home</a></li>
+      	<li class="mobile-list-item"><a href="./about.html">About</a></li>
+      	<li class="mobile-list-item"><a href="./#the-program">Program</a></li>
+      	<li class="mobile-list-item"><a href="./#speakers">Speakers</a></li>
+      	<li class="mobile-list-item"><a href="./about.html#partners">Sponsors</a></li>
+        <li class="mobile-list-item"><a href="#">Buy tickets now</a></li>
+      </ul>	
+    </nav>
+    <nav id="desktop-nav">
+    	<ul id="social-personal-links">
+    		<li><i class="fab fa-facebook-f"></i></li>
+    		<li><i class="fab fa-twitter"></i></li>
+    		<li>English</li>
+    		<li>My Page</li>
+    		<li>logout</li>
+    	</ul>	
+      <ul id="sections-pages-links">
+      	<li>
+      		<a href="./">
+      		  <img alt="logo" src="./images/general/event-logo.png" />
+      		</a>
+      	</li>
+      	<li><a href="./about.html">About</a></li>
+      	<li><a href="./#the-program">Program</a></li>
+      	<li><a href="./#speakers">Speakers</a></li>
+      	<li><a href="./#partners">Sponsors</a></li>
+        <li><a href="#">Buy tickets now</a></li>
+      </ul>	
+    </nav>	
+   </header>
+   <section id="about">
+  	 <p id="welcome-sentence">"Hello! Sharing world"</p>
+  	 <h1>The World’s Brightest Minds. One Unique Event 2022.</h1>
+  	 <hr>
+  	 <p id="mobile-description">
+	   GTC is a conference for all communities to engage with leading-edge AI and other groundbreaking technologies. These technologies affect everyone, and NVIDIA believes that diverse voices of the world should be heard. Our sessions and speakers are selected to inspire and impact developers from emerging countries, as well as historically underrepresented students and professionals.
+  	</p>
+  	<p id="desktop-description">
+	   GTC is a conference for all communities to engage with leading-edge AI and other groundbreaking technologies. These technologies affect everyone, and NVIDIA believes that diverse voices of the world should be heard. Our sessions and speakers are selected to inspire and impact developers from emerging countries, as well as historically underrepresented students and professionals.
+  	</p>
+  	<p id="contact-us">
+  	  For additional information about NVIDIA GTC or if you have any questions about event programming, sponsorships, registration, press/analyst lists, or general inquiries, please send an email to:
+  	  <span>GTC_registration@nvidia.com</span>
+  	  or call us at +1-408-486-2700.
+  	  Please allow one business day for a response.
+    </p>
+   </section>
+  </div>
+  <footer>
+	 <section id="previous-events">
+	 	<h2>See the past NVIDIA GTC</h2>
+	 	<hr>
+	 	<p>
+	 	  Take a look at the last NVIDIA GTC 
+	  </p>
+	  <div id="event1"></div>
+	  <div id="event2"></div>	 	
+	 </section>
```

Comment:
- [ ] [OPTIONAL] Hi, in this section there should be a text center over the two images. the year date for example can be centered on top of the image as per the reference design.

|Your Design|Reference Design|
-|-
|![image](https://user-images.githubusercontent.com/18730106/152521756-9cea1933-4b52-4f46-92a8-ea8155868f2a.png)|![image](https://user-images.githubusercontent.com/18730106/152521547-43950ad7-be27-4c73-8d32-8a098bcd305c.png)|